### PR TITLE
Agenda PDF

### PR DESCRIPTION
**Pull request by jsstevenson**

### CODE REVIEW

File: pamphlet/agenda.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+date: "September 2023"
+page-background: "img/gg-bridge.jpg"
+page-background-opacity: 0.1
+---
+
+### Sunday, September 17th
+
+| Time | Event |
+| -- | ------------------- |
+| 8:45 | Arrival &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; |
```
Comment (korikuzma) [inline pamphlet/agenda.md:13]: Can we include the comment about not arriving later than this time?

File: pamphlet/agenda.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+| 9:00 | Welcome and ice breaker |
+| 9:45 | Project presentations |
+| 10:15 | Form teams |
+| 10:30 | Hack! |
+| 12:00 | Lunch break |
```
Comment (korikuzma) [inline pamphlet/agenda.md:18]: Include [Delarosa Pizza](https://www.delarosasf.com/location/downtown/)

File: pamphlet/agenda.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+| 13:00 | Hack! |
+| 15:00	| Afternoon break |
+| 15:15 | Hack! |
+| 16:45 | Wrap up and early success stories |
+| 17:00 | Triage issues |
+| 18:00 | Transit |
+| 18:30 | Dinner at Chateau de Reece |
```
Comment (korikuzma) [inline pamphlet/agenda.md:25]: Include [Truly Mediterranean](https://www.trulyeats.com/catering-menu) wraps, platters, and kababs

File: pamphlet/agenda.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+### Monday, September 18th
+
+| Time | Event |
+| -- | ------------------- |
+| 8:45 | Arrival &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; |
+| 9:00 | Standup |
+| 9:45 | Hack! |
+| 12:30 | Lunch break |
```
Comment (korikuzma) [inline pamphlet/agenda.md:35]: Include [Uno dos tacos](https://order.toasttab.com/online/uno-dos-tacos)

**Review by korikuzma — CHANGES_REQUESTED** [review]

**Comment by korikuzma** [comment]
@reece You had made a note about too much going on at the end on day 1. What (if any) changes were you thinking?

File: pamphlet/agenda.pdf
Comment (korikuzma) [inline pamphlet/agenda.pdf:1]: Is it possible to make this one page? If we need to cut something, I think it's fine excluding the reason why we don't want people arriving late (can include this reason in slack channel)

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: pamphlet/agenda.pdf
Comment (jsstevenson) [reply pamphlet/agenda.pdf:1]: @korikuzma Adding the full "please arrive" text puts it onto two pages, I'll just do a short version

Comment (korikuzma) [reply pamphlet/agenda.pdf:1]: Sorry for all the requests. Can we have it be "latest time to arrive by" ?

Comment (jsstevenson) [reply pamphlet/agenda.pdf:1]: As in

```
| 8:45 | Arrival (latest time to arrive by) &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; |
```

?

Comment (korikuzma) [reply pamphlet/agenda.pdf:1]: I was just thinking this

```markdown
| 8:45 | Latest time to arrive by &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; |
```

Comment (korikuzma) [reply pamphlet/agenda.pdf:1]: Or maybe just changing it to like 8:55 for the start time? Idk

Comment (korikuzma) [reply pamphlet/agenda.pdf:1]: @reece what do you think?

**Comment by korikuzma** [comment]
Closing, we're moving to google doc with these changes

**Pull request closed without merging**